contracts/core/extensions/CoreAccounting.sol

```diff
@@ -117,31 +120,22 @@ contract CoreAccounting is
         external
         nonReentrant
     {
-        // Confirm an empty _tokens array is not passed
-        require(_tokens.length > 0, "WITHDRAW_TOKENS_ARRAY_EMPTY");
-
-        // Confirm an empty _quantities array is not passed
-        require(_quantities.length > 0, "WITHDRAW_QUANTITY_ARRAY_EMPTY");
-
-        // Confirm there is one quantity for every token address
-        require(_tokens.length == _quantities.length, "WITHDRAW_UNEQUAL_ARRAYS");
-
-        // For each token and quantity pair, run withdraw function
-        for (uint256 i = 0; i < _tokens.length; i++) {
-            withdrawInternal(
-                _tokens[i],
-                _quantities[i]
-            );
-        }
+        // Call internal batch withdraw function
+        batchWithdrawInternal(
+            msg.sender,
+            msg.sender,
+            _tokens,
+            _quantities
+        );
```

Comment:
Moved the existing batch withdraw to an internal function and call that function here